test/lsp.py

```diff
@@ -654,6 +654,9 @@ def user_interaction_failed_method_test(self, testcase, actual, expected):
                 "\nbut got:\n" + actual_pretty
             )
 
+        if self.suite.non_interactive:
+            return
```

Comment:
pylint seems right. This function normally returns `TestResult`. This would implicitly return `None`. Is `None` even a valid thing to return here?

Reply:
Ops, I probably was to hasty with my fix